local/php/envs.go

```diff
@@ -37,11 +37,30 @@ func (p *Server) generateEnv(req *http.Request) map[string]string {
 	}
 
 	pathInfo := req.URL.Path
+	filenameMatch := false
+
 	if pos := strings.Index(strings.ToLower(pathInfo), ".php"); pos != -1 {
 		file := pathInfo[:pos+4]
 		if _, err := os.Stat(filepath.Join(p.documentRoot, file)); err == nil {
 			scriptName = file
 			pathInfo = pathInfo[pos+4:]
+			filenameMatch = true
+		}
+
+	}
+
+	if pathInfo != string(os.PathSeparator) && filenameMatch == false {
+		paths := strings.Split(strings.TrimRight(pathInfo[1:], string(os.PathSeparator)), string(os.PathSeparator))
+		pathsLength := len(paths)
+		for i := 0; i < pathsLength; i++ {
+			poppedPath := paths[:pathsLength-i]
+			indexDir := string(os.PathSeparator) + filepath.Join(poppedPath...)
+			file := filepath.Join(indexDir, "index.php")
```

Comment:
what about using `p.passthru` here?
does it make sense?

Reply:
I don't think i understand it.

Reply:
ah ok. It might sense, might be sometimes weird with some complicated passthroughs, but it makes sense.